mtools/util/logevent.py

```diff
@@ -653,7 +749,23 @@ def _extract_counters(self):
                         try:
                             # Remap counter to standard name, if applicable
                             counter = counter_equiv.get(counter, counter)
-                            vars(self)['_' + counter] = int((token.split(':')
+                            if (counter == 'level' and token.startswith('level')):
+                                try:
+                                    self._readConcern = (
+                                    split_tokens[t + 1 + self.datetime_nextpos + 2].replace(',', ''))
```

Comment:
no need to split like this (unless it should be a tuple, but it doesn't look like it)

regardless, split_token should be one level down at least (if breaking the line like this)